modules/web/src/app/shared/components/cluster-summary/template.html

```diff
@@ -989,6 +989,10 @@ <h4>IPv6</h4>
                 <div key>Template ID</div>
                 <div value>{{machineDeployment.spec.template.cloud?.anexia?.templateID}}</div>
               </km-property>
+              <km-property>
+                <div key>Template Name</div>
+                <div value>{{machineDeployment.spec.template.cloud?.anexia?.template}}</div>
+              </km-property>
```

Comment:
We should remove `Disk Size in GB` and display added disks.
Also, we should display only one of Template ID or Template Name (depends which one has value) since both cannot have values together.